internal/utils/util.go

```diff
@@ -19,7 +19,7 @@ func ParseCSV(r io.Reader, batchSize int) ([][]*model.Student, error) {
 	dataRows := rows[1:]
 
 	// Map the header names to their column indexes
-	var nameIndex, classIndex, birthdayIndex int
+	var nameIndex, classIndex, birthdayIndex, examGroupIndex int
 	for i, col := range header {
 		switch col {
 		case "name":
```

Comment:
shouldn't hard code in here => we should create constant file and put it in file